scripts/edit_media_directories.py

```diff
@@ -5,11 +5,22 @@
 import requests
 import logging
 import sys
+import subprocess
+import urllib3
+import urllib3.exceptions
+
+def CheckLMSMode():
```

Comment:
CamelCase is typically used only for classes in python. Is there a specific reason this and the LMSMode variable below are in camel case?

Reply:
CamelCase is simply how I write code, back when I was taught in school originally that's how we did everything and that practice has stuck with me